app/src/oculusvr/cpp/DeviceDelegateOculusVR.cpp

```diff
@@ -1110,6 +1110,30 @@ DeviceDelegateOculusVR::RegisterImmersiveDisplay(ImmersiveDisplayPtr aDisplay) {
   m.UpdatePerspective();
 }
 
+void
+DeviceDelegateOculusVR::SetImmersiveSize(const uint32_t aEyeWidth, const uint32_t aEyeHeight) {
+  uint32_t recommendedWidth, recommendedHeight;
+  m.GetImmersiveRenderSize(recommendedWidth, recommendedHeight);
+
+  const uint32_t maxWidth = recommendedWidth * 2;
+  const uint32_t maxHeight = recommendedHeight * 2;
+  const uint32_t minWidth = recommendedWidth / 2;
+  const uint32_t minHeight = recommendedHeight / 2;
+
+  const uint32_t targetWidth = (uint32_t) fmaxf(fminf(aEyeWidth, maxWidth), minWidth);
+  const uint32_t targetHeight = (uint32_t) fmaxf(fminf(aEyeHeight, maxHeight), minHeight);
+
```

Comment:
nit: I am thinking if we can move 
```
   const uint32_t maxWidth = recommendedWidth * 2;
   const uint32_t maxHeight = recommendedHeight * 2;
   const uint32_t minWidth = recommendedWidth / 2;
   const uint32_t minHeight = recommendedHeight / 2;
   const uint32_t targetWidth = (uint32_t) fmaxf(fminf(aEyeWidth, maxWidth), minWidth);
   const uint32_t targetHeight = (uint32_t) fmaxf(fminf(aEyeHeight, maxHeight), minHeight);
```
to class DeviceDelegate as `GetTargetWidth()` and `GetTargetHeight()`, And getting them via `uint32 w = GetTargetWidth(recommendedWidth);`, `uint32 h = GetTargetHeight(recommendedHeight);`  how do you think about it ?

Reply:
Done